### ISSUE TITLE
Vis2 error

### ISSUE THREAD

**Issue by reaperisonline** [iseahound/Vis2#14]
Hello,

Let me start by saying I want to honestly say that I am not using Vis2 through AHK, but through Pullover Macro Creator. I know how annoying it can be to receive a help request from something completely different than what you developed it for, so i would like to apologize in advance, but unfortunately I have run into a complete dead end and reaching out to you is my last resort to figure out what the issue is and (if even possible) solve it by circumventing the programming. Even the developer of PMC as well as our own IT department/company security has no clue how to solve this.

The function used was OCR /  'read text from screen and convert to variable'

Have you seen the error below before? Could you enlighten me on what it means and if there is a way to solve this? It seems like PMC/Vis2 is trying to save a temporary file in a restricted location. The computer i am trying to get this to work on is heavily secured. It seems to work 'okay' in admin mode, but this is not a long term solution since the user can not have access to this and needs to call IT support everytime for a temporary fix. If you could enlighten me on the issue, that would already help a bunch so we might be able to program around the security issue.

![image](https://user-images.githubusercontent.com/47637050/153041972-69ceea00-8fb2-41c3-8dad-c76d8101e656.png)

**Comment by iseahound** [comment]
Yes, it should be possible. I'll have to add it in a later update.
